src/test/java/duke/command/DeleteCommandTest.java

```diff
@@ -0,0 +1,90 @@
+package duke.command;
+
+import duke.Storage;
+import duke.Ui;
+import duke.task.EventStub;
+import duke.task.TaskList;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.fail;
+
+
+class DeleteCommandTest {
+    TaskList tasks;
+    Ui ui;
+    Storage storage;
+    ByteArrayOutputStream output;
+    String saveFile = "test.txt";
+    String fileSeparator = File.separator;
+    // Map project path to the directory from which you run your program
+    String projectRootPath = Paths.get("").toAbsolutePath().toString();
+    String dataDirectoryPath = projectRootPath + fileSeparator + "data";
+    String saveFilePath = dataDirectoryPath + fileSeparator + saveFile;
+    static final String doneStatusIcon = "\u2713"; // Check mark icon
+    static final String incompleteStatusIcon = "\u2718"; // Cross mark icon
+    EventStub eventTask = new EventStub("project meeting", "2020-01-01",
+            false, "event,0,project meeting,2020-01-01",
+            doneStatusIcon, incompleteStatusIcon,
+            "[E][" + incompleteStatusIcon + "] project meeting " + "(at: Jan 1 2020)");
+
+    void deleteSaveFile() {
+        try {
+            Files.deleteIfExists(Paths.get(saveFilePath));
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
In general, printing stack trace is not good error handling. Think if there is anything better you can do about the exception rather than printing the stack trace.